Sources/WoosmapGeofencing/Business Logic/Distance.swift

```diff
@@ -142,7 +143,13 @@ public class Distances {
                     }
                 }
             } else {
-                print("WoosmapGeofencing.DistanceAPIData " + jsonStructure.status!)
+                if(WoosLog.isValidLevel(level: .info)){
```

Comment:
These if conditions are repeated many times? Can't they be put in a single function?

Reply:
it is not in loop